CMakeLists.txt

```diff
@@ -15,6 +15,12 @@
 # limitations under the License.
 #
 cmake_minimum_required(VERSION 3.21)
+
+if(WIN32)
+    include_directories(${CUDA_TOOLKIT_ROOT_DIR}/include)
+    link_directories(${CUDA_TOOLKIT_ROOT_DIR}/lib/x64)
+endif(WIN32)
```

Comment:
This shouldn't be necessary. TC_CORE already links against cuda::cudart from CMake's FindCudaToolkit https://cmake.org/cmake/help/latest/module/FindCUDAToolkit.html. If a toolkit component is missing for one cmake target it should use only link cuda::xxx for that specific target and not globally

Reply:
you are right, this change is not required as it is for my local setup

Reply:
we should really automate this. e.g. via https://github.com/stefanzweifel/git-auto-commit-action

Reply:
will be reverted through Commit 697717f7